src/main/java/Duke.java

```diff
@@ -1,10 +1,223 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args)  {
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+
+        String userInput;
+        Scanner in = new Scanner(System.in);
+        Task[] userTasks = new Task[0];
+        Boolean isContinue = true;
+        while (isContinue) {
+            userInput = in.nextLine();
+            userInput = userInput.trim();
+            String[] userCommands = userInput.split(" ");
+            String userCommand = userCommands[0];
+            try {
+                switch (userCommand) {
+                case "list":
+                    userCommandList(userTasks);
+                    break;
+                case "bye":
+                    userCommandBye();
+                    isContinue = false;
+                    break;
+                case "mark":
+                    userCommandMark(userTasks, userCommands);
+                    break;
+                case "unmark":
+                    userCommandUnmark(userTasks, userCommands);
+                    break;
+                case "todo":
+                    userTasks = userCommandTodo(userTasks, userCommand, userInput);
+                    break;
+                case "deadline":
+                    userTasks = userCommandDeadline(userTasks, userCommand, userInput);
+                    break;
+                case "event":
+                    userTasks = userCommandEvent(userTasks, userCommand, userInput);
+                    break;
+                default:
+                    userCommandDefault();
+                    break;
+                }
+            } catch (DukeException e) {
+                ;
+            }
+        }
+    }
+
+    private static Event getNewEventTask(String userInput, String userCommand) throws DukeException {
+        String taskString = getTaskString(userInput,userCommand);
+        String taskName = getEventTaskName(taskString);
+        String eventFromDate = getEventFromDate(taskString);
+        String eventToDate = getEventToDate(taskString);
+        Event newEventTask = new Event(taskName,eventFromDate,eventToDate);
```

Comment:
Within your instantiation of Event, remember to add a spacing in between each inputs. And do the same for the rest of your code.